src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    
+    public String value;
+    public boolean isPrivate;
+
+    /**
+     * Returns true if the given string is a valid person email.
+     */
+    public static boolean isValid(String test, String regex) {
```

Comment:
Good job extracting this into a method! Do you think it can be abstracted even further?